recipe/meta.yaml

```diff
@@ -17,6 +17,7 @@ build:
 requirements:
   build:
     - cmake >=3.12
+    - make
```

Comment:
This is literally the only manual change, everything else is the work of conda-forge's automatic migrator

Reply:
@stepanhruda pls ping @tstenner and @conda-forge/core to merge